wordplay.py

```diff
@@ -222,7 +226,7 @@
 # Question: 
 # What are all of the letters that never appear consecutively in an English word? For example, we know that “U” isn’t an answer, because of the word VACUUM, and we know that “A” isn’t an answer, because of “AARDVARK”, but which letters never appear consecutively?
 
-alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
```

Comment:
Annoyingly in this case I'd say alphabet isn't actually a constant because it's being reassigned here.

I think that it makes sense to make `alphabet` itself a constant, but perhaps create a variable that initially has the value of alphabet that gets modified in here?

For example: 

```python
letters_with_no_consecutive_appearances = ALPHABET
# ...
letters_with_no_consecutive_appearances = letters_with_no_consecutive_appearances.replace(word[word_index], '')
```

Or something along those lines. Don't stress it though - correctness is the most important thing and I believe you've got that covered 👍